### PR TITLE
#164368096 create custom button component

### PR DESCRIPTION
**Pull request by bobbylemm**
#### What does this PR do?
Create a custom button component.
#### How should this be manually tested?
Clone the Repo, run `npm install`, then run `npm test`.
#### What are the relevant pivotal tracker stories?
[#164368096](https://www.pivotaltracker.com/story/show/164368096)
#### Screenshots (if appropriate)

screenshot 2019-03-07 at 1 43 37 pm

### CODE REVIEW

**Review by ozone4real — DISMISSED** [review]
LGTM

**Review by biolabalo — DISMISSED** [review]

**Review by omenkish — DISMISSED** [review]

**Review by EfeAgare — DISMISSED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 language: node_js
 node_js:
- - "stable"
+  - "11.10.1"
```
Comment (p-funky) [inline .travis.yml:3]: I'm curious about this change. Was there a particular reason?

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "the front-end part of the authors haven application",
   "main": "index.js",
   "scripts": {
-    "test": "jest \"--coverage\"",
+    "test": "jest \"--coverage\" \"-u\"",
```
Comment (p-funky) [inline package.json]: Is the purpose of adding the `-u` flag for updating snapshots? If yes, then there should be a separate command for that actually.

File: src/styles/styledComponents/Button.jsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  border: ${props => (props.danger ? '0.12rem solid #CD160B' : '0.12rem solid #0B41CD')};
+  margin: 1rem;
+  text-transform: capitalize;
+  transition: all 0.3s;
+  height: 35px;
+  width: 97px;
+  display: ${props => (props.inlineButton ? 'inline-block' : 'block')};
+`;
+
+export default Button;
```
Comment (p-funky) [inline src/styles/styledComponents/Button.jsx:25]: So is it possible at all to pass `style` to this button component?

Comment (bobbylemm) [reply src/styles/styledComponents/Button.jsx:25]: Yes it is very possible

File: .travis.yml (hunk 1 above)
Comment (bobbylemm) [reply .travis.yml:3]: There was an error with Travis when it tried to run the test, and after researching about the problem. I found a solution to downgrade the node version from `11.11.1` to `11.10.1`. It turned out Travis had a problem with this node version when It came to testing react components

File: package.json (hunk 2 above)
Comment (bobbylemm) [reply package.json]: Ok, but wouldn't there be an issue with how the test runs. Since there are now two test commands

**Review by biolabalo — APPROVED** [review]

**Review by primuse — APPROVED** [review]

**Pull request merged**